docs/docs/upgrading.md

```diff
@@ -9,10 +9,22 @@ description: >-
 
 ## Since 0.3.0
 
-### Breaking: No default certificate location
+### Breaking
```

Comment:
Should the removal of HTTP health check for authorize be flagged here?

Reply:
Updated